config/paths.php

```diff
@@ -48,6 +48,8 @@
 
 /**
  * File path to the webroot directory.
+ * To derive your webroot from your webserver change this to:
+ * `define('WWW_ROOT', rtrim($_SERVER['DOCUMENT_ROOT'], DS) . DS);`
```

Comment:
When would this be necessary?

Reply:
The rtrim is because I do not trust the webserver to have the trailing slash.
However the DOCUMENT_ROOT was necessary when I create a cakephp project that had multiple webroots (portalling/sharding) but shares the same application structure.

Reply:
I don't think we need to add comments for such edge case usages. It would unnecessarily confuse users.